docs/services/azure-services-guide.md

```diff
@@ -0,0 +1,625 @@
+sqlserverclass.database.azure.crossplane.io/azure-mysql-standard created
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get sqlserverclasses -n azure-infra-dev
+```
+*Output*
+```bash
+NAME                   PROVIDER-REF   RECLAIM-POLICY   AGE
+azure-mysql-standard   demo-azure     Delete           11s
+```
+
+You are free to create more Azure `SQLServerClass` instances to define more
+potential configurations. For instance, you may create `large-azure-mysql` with
+field `storageGB: 100`.
+
+#### Application Namespaces
+Earlier, we created a namespace to group our Azure infrastructure resources.
+Because our application resources may be satisfied by services from any cloud
+provider, we want to separate them into their own namespace. For this demo, we
+will create a namespace called `app-project1-dev`, which we will use to group
+our Wordpress resources.
+
+* Define a `Namespace` in `app-project1-dev-namespace.yaml`:
+```yaml
+cat > app-project1-dev-namespace.yaml <<EOF
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: app-project1-dev
+EOF
+```
+
+* Create the `Namespace`:
+```bash
+kubectl create -f app-project1-dev-namespace.yaml
+```
+
+* You should see the following output:
+```bash
+namespace/app-project1-dev created
+```
+
+#### Portable Resource Classes
+Portable resource classes are used to define a class of service in a single
+namespace for an abstract service type. We want to define our Azure
+`SQLServerClass` as the standard MySQL class of service in the namespace that
+our Wordpress resources will live in.
+
+* Define a `MySQLInstanceClass` in `mysql-standard.yaml` for namespace
+  `app-project1-dev`:
+```yaml
+cat > mysql-standard.yaml <<EOF
+---
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstanceClass
+metadata:
+  name: mysql-standard
+  namespace: app-project1-dev
+classRef:
+  kind: SQLServerClass
+  apiVersion: database.azure.crossplane.io/v1alpha2
+  name: azure-mysql-standard
+  namespace: azure-infra-dev
+EOF
+```
+
+* Create the `MySQLInstanceClass`:
+```bash
+kubectl apply -f mysql-standard.yaml
+```
+
+* You should see the following output:
+```bash
+mysqlinstanceclass.database.crossplane.io/mysql-standard created
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get mysqlinstanceclasses -n app-project1-dev
+```
+*Output*
+```bash
+NAME             AGE
+mysql-standard   27s
+```
+
+Once again, you are free to create more `MySQLInstanceClass` instances in this
+namespace to define more classes of service. For instance, if you created
+`mysql-azure-large` above, you may want to create a `MySQLInstanceClass` named
+`mysql-large` that references it. You may also choose to create MySQL resource
+classes for other non-Azure providers, and reference them for a class of service
+in the `app-project1-dev` namespace.
+
+You may specify *one* instance of a portable class kind as *default* in each
```

Comment:
I am not sure I would understand this as a new user. Maybe it'd make sense to tell the user how to add the `label` if they're interested, and then link them to a document which explains the concepts in more depth?

Reply:
I believe this is demonstrated in the following lines after this one